packages/extension-text-align/src/text-align.ts

```diff
@@ -41,10 +41,6 @@ export const TextAlign = Extension.create<TextAlignOptions>({
             default: this.options.defaultAlignment,
             parseHTML: element => element.style.textAlign || this.options.defaultAlignment,
             renderHTML: attributes => {
-              if (attributes.textAlign === this.options.defaultAlignment) {
```

Comment:
I think the idea here was to not add any style attributes to the element when the alignment is equal to the default alignment.

If we remove this, there's no way to get rid of alignment inline styles without removing a node first even though they are mostly not needed.

I think the only issue that could come up with this is, that the text alignment can diverge from the default because of a wrapping element that updates the text alignment and passes it down via inheritance but this is expected browser behaviour I'd say?

Open for discussion on this one.